lib/circuitbox/circuit_breaker.rb

```diff
@@ -26,7 +26,9 @@ def initialize(service, options = {})
       @service = service
       @circuit_options = options
       @circuit_store   = options.fetch(:cache) { Circuitbox.circuit_store }
-      @notifier        = options.fetch(:notifier_class) { Notifier }
+
+      notifier_class = options.fetch(:notifier_class) { Notifier }
```

Comment:
This would be an interesting thing to do. I think we get some benefits where we could have ```default_notifier``` smart enough to load ActiveSupport notifier or one that just doesn't do any logging.